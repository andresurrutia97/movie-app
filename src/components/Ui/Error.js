import React from 'react';

const Error = () => (
  <p className="w-full py-5 text-center">Something went wrong</p>
);

export default Error;
