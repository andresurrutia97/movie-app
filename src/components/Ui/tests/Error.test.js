import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Error from '../Error';

describe('<Error/>', () => {
  const component = <Error />;

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a <p> tag with a text equal to "Something went wrong"', () => {
    const wrapper = shallow(component);
    expect(wrapper.props().children).toBe('Something went wrong');
  });
});
