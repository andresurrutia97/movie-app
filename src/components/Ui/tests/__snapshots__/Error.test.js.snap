// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error/> should render correctly  1`] = `
<p
  className="w-full py-5 text-center"
>
  Something went wrong
</p>
`;
